.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -579,6 +579,9 @@ AddType text/vtt                            vtt
     RewriteRule ^dedicated/support/summarizing-cluster-specifications.html /dedicated/osd_support/osd-summarizing-cluster-specifications.html [NE,R=301]
     RewriteRule ^dedicated/support/?(.*)$ /dedicated/osd_support/$1 [NE,R=301]
 
+    # Redirects for PTP changes delivered in https://github.com/openshift/openshift-docs/pull/35889
+    RewriteRule ^networking/configuring-ptp.html /networking/using-ptp.html [NE,R=302]
```

Comment:
@bscott-rh Similar to my comment on the other PR, do we need `^(container-platform)/(4\.10|4\.11)` bit here as well?

Reply:
updated

Reply:
https://htaccess.madewithlove.com/

Reply:
@bscott-rh @gaurav-nelson This one can be merged now if it looks good btw. redirect should have been done months ago...